### PR TITLE
Move DrawMenuBar to Interop User32

### PR DESCRIPTION
**Pull request by gpetrou**
## Proposed changes

- Move DrawMenuBar to Interop User32.

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/dotnet/winforms/pull/2616)

### CODE REVIEW

**Comment by RussKie** [comment]
MC

**Comment by hughbe** [comment]
One question, probably for @RussKie: do we need this code now that we've removed support for `Menu`

**Comment by RussKie** [comment]
> do we need this code now that we've removed support for `Menu`

That's a very good questions, thank you for raising it!

I had a very brief look and it looks like this and few other methods are now redundant. Further investigation is required however.

**Comment by gpetrou** [comment]
@RussKie what should we do about this one? Remove the methods instead?

**Comment by RussKie** [comment]
Looks like we still have code paths that execute the method. However they badly need some cleaning, e.g.
https://github.com/dotnet/winforms/blob/e5484a6448bfa94a236a8d1296d6e9d60b774b04/src/System.Windows.Forms/src/System/Windows/Forms/Form.cs#L5668-L5689

The overload for `UpdateMenuHandles` is only called with `forceRedraw: true`, so the method can be inlined in the default method, and `forceRedraw` removed completely.
https://github.com/dotnet/winforms/blob/e5484a6448bfa94a236a8d1296d6e9d60b774b04/src/System.Windows.Forms/src/System/Windows/Forms/Form.cs#L5691

@gpetrou do you think you can do it as part of this PR?

File: src/System.Windows.Forms.Primitives/src/Interop/User32/Interop.DrawMenuBar.cs [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+using System;
+using System.Runtime.InteropServices;
+
+internal static partial class Interop
+{
+    internal static partial class User32
+    {
+        [DllImport(Libraries.User32, ExactSpelling = true)]
+        public static extern BOOL DrawMenuBar(IntPtr hWnd);
```
Comment (RussKie) [inline src/System.Windows.Forms.Primitives/src/Interop/User32/Interop.DrawMenuBar.cs]: There are no callers for this one, so there is no reason for it to be public.

**Comment by RussKie** [comment]
Thank you

**Pull request merged**